lib/db.js

```diff
@@ -81,4 +81,82 @@ const examplegames = [
+    image: "/assets/backhand_female.jpg",
+    alt: "Backhand lesson",
+    step1: "Racket up",
+    step2: "Unit turn",
+    step3: "C Shape",
+    step4: "End swing path",
+  },
+  {
+    id: "3",
+    name: "slice",
+    description:
+      "A shot that imparts backspin on the ball by hitting the ball with a high-to-low motion.",
+    image: "/assets/backhand_slice_male.jpg",
+    alt: "Slice lesson",
+    step1:
+      "The ideal grip for a one-handed slice is the continental grip, the same grip you use to serve",
+    step2:
+      "All slice backhands are hit with a closed stance, stepping in the direction of the shot or slightly across the body.",
+    step3:
+      "As with a topspin backhand you have to take the racquet back using the upper body and not just the arm. Your non-dominant hand should remain on the throat of the racquet. Your hitting arm should remain fairly straight and the racquet head open. As you swing forward your racquet will close and at contact it will be almost parallel to the net.  Your upper body remains sideways on this groundstroke.",
+    step4: "End swing path",
+  },
+
+  {
+    id: "4",
+    name: "serve",
+    description:
+      "A serve (or, more formally, a service) in tennis is a shot to begin the point. The most common serve is used is an overhead serve. It is initiated by tossing the ball into the air over the server's head and hitting it when the arm is fully stretched out (usually near the apex of its trajectory) into the diagonally opposite service box without touching the net. The server may employ different types of serve: a flat, a kick, or a slice serve.",
+    image: "/assets/serve_male.jpg",
+    alt: "Serve lesson",
+    step1: "Racket up",
+    step2: "Unit turn",
+    step3: "C Shape",
+    step4: "End swing path",
+  },
+  {
+    id: "5",
+    name: "volley",
+    description:
+      "A volley is made in the air before the ball bounces, generally near the net or inside of the service line. Volleys consist of the forehand volley and backhand volley[2] and are usually made with a stiff-wristed motion to hit the ball into an open area of the opponent's court. The half volley is made by hitting the ball on the rise just after it has bounced, once again generally in the vicinity of the net.",
+    image: "/assets/volley_male.jpg",
+    alt: "Volley lesson",
+    step1: "Racket up",
+    step2: "Unit turn",
+    step3: "C Shape",
+    step4: "End swing path",
+  },
+];
+
 export { examplegames };
+
+export async function getAllLessons() {
+  return lessons;
+}
+
+export async function getLessonByName(name) {
+  return lessons.find((lesson) => {
+    return lesson.name === name;
+  });
+}
```

Comment:
Hey Phillip, I think these functions do not have to be async since you don't fetch the data from a real db.